Importer/Interpreter/Field/DatetimeFieldInterpreter.php

```diff
@@ -38,4 +41,18 @@ public function interpret(FieldConfigurationInterface $configuration, $value)
 
         return $dateTime;
     }
+
+    /**
+     * @param string $value
+     *
+     * @return bool
+     */
+    private function containsTime($value)
+    {
+        if (preg_match("/([0-2]+[0-9]):([0-5]?[0-9]):([0-5]?[0-9])$/", $value)) {
```

Comment:
Do we want to handle times without seconds?

Might be good idea to add a unit test for this.

You don't have to create groups if you don't use them.
What if there is a space character at the end?
"200000000:0:0" - what kind of time is it?